packages/parser/tests/unit/envelopes/kinesis-firehose.test.ts

```diff
@@ -24,9 +29,10 @@ describe('Kinesis Firehose Envelope', () => {
 
     it('should parse a single record for SQS event', () => {
       const mock = generateMock(TestSchema);
-      const testEvent = TestEvents.kinesisFirehoseSQSEvent as z.infer<
-        typeof KinesisFirehoseSchema
-      >;
+      const testEvent = getTestEvent<KinesisFireHoseSqsEvent>({
```

Comment:
As we discussed in the catch up, I suggest we move these I/o operations in the main `describe` block and we reuse the events by cloning/extending them, similar to what done [here](https://github.com/aws-powertools/powertools-lambda-typescript/blob/ec11b4ec0d71fed04a7debc00630bed16fadd67b/packages/parser/tests/unit/schema/kinesis-firehose.test.ts#L10-L21).